java/connector-node/risingwave-sink-jdbc/src/main/java/com/risingwave/connector/JDBCSink.java

```diff
@@ -92,8 +94,23 @@ private PreparedStatement prepareStatement(SinkRow row) {
                 try {
                     PreparedStatement stmt =
                             conn.prepareStatement(insertStmt, Statement.RETURN_GENERATED_KEYS);
+                    var columnNames = getTableSchema().getColumnNames();
                     for (int i = 0; i < row.size(); i++) {
-                        stmt.setObject(i + 1, row.get(i));
+                        switch (getTableSchema().getColumnType(columnNames[i])) {
+                            case INTERVAL:
+                                stmt.setObject(i + 1, new PGInterval((String) row.get(i)));
+                                break;
+                            case JSONB:
```

Comment:
Are the added data types only workable for PG? Or is it generally workable for jdbc compatible database? Or at least also workable for MySQL?

Reply:
Those objects should only work in PG since they are provided by the PG driver. I'll add a check for the downstream type.